src/index.js

```diff
@@ -43,7 +43,8 @@ function makeLoader(callback) {
 
     loader
       .call(this, source, inputSourceMap, overrides)
-      .then(args => callback(null, ...args), err => callback(err));
+      .then(args => callback(null, ...args))
+      .catch(err => callback(err));
```

Comment:
While theoretically pretty similar, this is technically worse because it means that if `callback(null, ...args)` threw an exception, it would call `callback(err)`, meaning `callback` would get called with an error that it actually threw itselff, instead of triggering an uncaught error.